From 3c3e2f983391913891979aed64ed563f20a34c83 Mon Sep 17 00:00:00 2001
From: Jacopo Mondi <jacopo.mondi@ideasonboard.com>
Date: Thu, 29 Feb 2024 09:32:34 +0100
Subject: [PATCH 0919/1085] media: ov64a40: Use container_of_const()

The container_of_const(), introduced in Linux v6.2, allows to preserve
the const-ness of a pointer.

Use it in the sd_to_ov64a40() function.

Signed-off-by: Jacopo Mondi <jacopo.mondi@ideasonboard.com>
---
 drivers/media/i2c/ov64a40.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/media/i2c/ov64a40.c
+++ b/drivers/media/i2c/ov64a40.c
@@ -2858,7 +2858,7 @@ struct ov64a40 {
 
 static inline struct ov64a40 *sd_to_ov64a40(struct v4l2_subdev *sd)
 {
-	return container_of(sd, struct ov64a40, sd);
+	return container_of_const(sd, struct ov64a40, sd);
 }
 
 static const struct ov64a40_timings *
